### DISCUSSION TITLE
Cần chỉnh sửa về giao diện trang chủ

### DISCUSSION THREAD

**Discussion by ankieu123** [dinhgiaandev/CodeGuru_BE#45] [General]
- Màu sắc của giao diện phần Night mode hơi bị tối.
- Font chữ nên thay đổi sao cho hợp với chữ tiếng Việt

**Comment by dinhgiaan** [comment]
vậy có cần đổi bố cục giao diện hay gì luôn không

**Reply by ankieu123** [reply]
Bố cục hiện tại ok r, để thời gian tới coi có cần đổi k

**Reply by dinhgiaan** [reply]
ok fen 👌
